model/Core/Classes/CreationInfo.md

```diff
@@ -10,6 +10,8 @@ Provides information about the creation of the Element.
 
 The CreationInfo provides information about who created the Element, and when and how it was created. 
 
+The dateTime created is often the date of last change (e.g., a git commit date), not the date when the SDPX data was created, as doing so supports reproducible builds.
```

Comment:
SDPX => SPDX

Reply:
Whups! Thanks for pointing out the typo!